queries.sql

```diff
@@ -8,3 +9,45 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <=17.3;
+
+/* Milestone 2*/
+
+-- Transaction 1 
+BEGIN;
+UPDATE animals SET species='unspecified';
+ROLLBACK;
+
+-- Transaction 2
+BEGIN;
+UPDATE animals SET species='digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species='pokemon' WHERE species IS NULL;
+COMMIT;
```

Comment:
- [ ] Before COMMIT kindly consider verifying if the changes are made successfully. you can add `SELECT species from animals;` before and after the `COMMIT`

Reply:
HI @Ismailco, I did it you can see it in the screenshot that I put in the PR.